internal/resources/resource_volume.go

```diff
@@ -211,6 +213,22 @@ func volumeSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "The number of active exports for this volume",
 		},
+
+		vDataCenterName: {
+			Type:        schema.TypeString,
+			Required:    false,
+			Optional:    false,
```

Comment:
shouldn't one of these be true?

Reply:
I will remove this Optional parameter as it is a read only field.